github-flow.rst

```diff
@@ -0,0 +1,153 @@
+GitHub
+======
+
+Proces komunikacji na GitHubie powinien zakładać jak największe zaufanie
+i odpowiedzialność jej uczestników. Wpłynie to pozytywnie na ich
+zaangażowanie, a w konsekwencji na cały proces komunikacji.
+
+Publikujemy pod swoim nazwiskiem
```

Comment:
Jasne, po angielsku byłoby to po prostu "under one's own name", po polsku "autor" jest zgrabniejsze. Dziękuję :)